src/component/component/index.js

```diff
@@ -137,19 +137,31 @@ export class Component extends BaseComponent {
         // Register all of the drivers for instantiating components. The model used is -- there's a standard javascript
         // way of rendering a component, then each other technology (e.g. react) needs to hook into that interface.
         // This makes us a little more pluggable and loosely coupled.
-
+        this.driverResults = {};
```

Comment:
Its possible, actully I would have prefered that, however, it can be considered a breaking change scince same function returns the component itself in other drivers. i.e. angularJS